lib/registration.pm

```diff
@@ -220,6 +222,32 @@ sub register_addons_cmd {
     }
 }
 
+sub check_registered_system {
+    my ($system) = @_;
+    my $version = get_var('SLE_PRODUCT') . $system;
+    assert_script_run "zypper lr --uri | grep -i $version";
+}
+
+sub check_registered_addons {
+    my ($addonlist) = @_;
+    $addonlist //= get_var('SCC_ADDONS');
+    if (is_sle('>15')) {
+        $addonlist = $addonlist . ",base,desktop,sdk,lgm,python2,serverapp,wsm";
+    }
+    my @addons = grep { defined $_ && $_ } split(/,/, $addonlist);
+    foreach my $addon (@addons) {
+        my $name = get_addon_fullname($addon);
+        if ($name =~ /LTSS/) {
+            $name = 'LTSS';
+        }
+        assert_script_run "zypper lr --uri | grep -i $name";
+        # If has WE addon, need check nvidia repo
```

Comment:
I suggest to add the poo number here.

Reply:
We don't have this poo, just talking with gaowei